src/map/MapSidebar.js

```diff
@@ -310,6 +311,24 @@ function MapSidebarMain (props) {
     }
   }, [api, cid, participantCount]);
 
+  const [nextMeetupTime, setNextMeetupTime] = useState([]);
+  // gets the date of the next Meetup
+  useEffect(() => {
+    async function getNextMeetupDate () {
+      const tilebelt = require('@mapbox/tilebelt');
+      const timespace = require('@mapbox/timespace');
+      const meetupLocations = await api.rpc.encointer.getLocations(cid);
+      const tempLocation = locationFromJson(api, meetupLocations[0]);
+      const tile = tilebelt.pointToTile(meetupLocations[0].lon, meetupLocations[0].lat, 8);
+      const temptimespace = timespace.getFuzzyTimezoneFromTile(tile);
+      const tempTime = await getNextMeetupTime(api, tempLocation);
+      debug && console.log('the date is: ' + formatDate(tempTime.toNumber()));
+      const nextMeetupDate = formatDate(tempTime.toNumber()).split(',');
```

Comment:
this will not work AFAIU. formatDate applies the locale where the browser is located. that's why it looks ok for Zurich as I'm calling the explorer from zurich